common/frame_session.go

```diff
@@ -745,7 +751,7 @@ func (fs *FrameSession) onAttachedToTarget(event *target.EventAttachedToTarget)
 	session := fs.page.browserCtx.conn.getSession(event.SessionID)
 	targetID := event.TargetInfo.TargetID
 
-	if event.TargetInfo.Type == "iframe" {
+	if eti := event.TargetInfo; eti.Type == "iframe" && eti.URL != "" {
```

Comment:
I question whether this is something we want to do (Playwright doesn't for example), discarding iframe target attachments because URL is empty. Why is this problematic? Iframes can be created without a `src` attribute (will load `about:blank`) and then have the `src` set programmatically, and if you remove the `src` attribute it will also automatically load `about:blank` according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

Reply:
> Why is this problematic?

I wonder why it solves most of the issues when we do this? 🤷  I'm open to any help. I feel like I'm stuck, and it seems like I can't see any other solutions for this issue 😓 

I guess this fixes most of the issues because our system is not a 100% match of Playwright, so ours is acting differently.  I suggest merging and keeping this fix for a while and seeing it in action. WDYT?

> I'd suggest running the equivalent of Tom's script in Playwright (I did that a lot with other scripts, analyzing the CDP traffic differences with xk6-browser).

I wrote a script to run in Playwright, and it worked. I'll keep analyzing the CDP traffic differences.

> Looks mostly good, great that you fix all my go noob mistakes (`make()` etc.) 🙌😅

You're welcome! ❤️

Reply:
> Iframes can be created without a src attribute (will load about:blank) and then have the src set programmatically, and if you remove the src attribute it will also automatically load

Maybe this is the reason why skipping frames with empty src works.